summary/summary.go

```diff
@@ -83,15 +84,15 @@ func Summary(c *k8s.Client, o Options) error {
 			Aggregate: o.Aggregation,
 		})
 		if err != nil {
-			return err
+			return nil, err
 		}
 		DisplaySummaryOutput(sumResp, o.RevDNSLookup, o.Type)
 
```

Comment:
there is one bug already here
if summary is to be filtered based on given pod name and output format is json then we'll not get desired output.
can you handle this case too. and we need to return the result too if the format is json.

Reply:
@rksharma95 updated, please check